package/libskylark/docs/about.md

```diff
@@ -0,0 +1,88 @@
+# Skylark Upload and Download Daemons
+
+libskylark is a curl-based library used for publishing and subscribing SBP data
+via Piksi's Linux core. We use it in two SBP-over-HTTP daemon services that can
+be optionally enabled and monitored on the Piksi: `skylark_upload_daemon` and
+`skylark_download_daemon`.
+
+## Why
+
+Provide direct network connectivity to Skylark services. Historical connectivity
+has been provided through the serial port either through the Piksi Console or
+other utilities. Moving to a direct connection to Skylark services will remove a
+lot of troublesome intermediaries, especially the Piksi Console, and hopefully
+provide simpler and more robust network connectivity. This will completely
+remove the need for external, host-provided network connectivity.
+
+## How
+
+Setup upload and download daemons that are monitored and continuously run on
+Piksi's Linux core - if either process crashes, they will be restarted by
+monit. The upload and download daemons run independently for improved robustness
+and simplicity, pulling and pushing SBP data to two Skylark-specific ZeroMQ
+ports that are exposed on the Linux host on are routed to the firmware:
+`tcp://127.0.0.1:43070` and `tcp://127.0.0.1:43071`, respectively. Taken
```

Comment:
@mfine if you update the port numbers, the init.d/monit configuration, or the command line flags, please update the document.